cmd/info.go

```diff
@@ -90,30 +89,46 @@ func (o *InfoOptions) Run() error {
 		}
 	}
 
+	if len(o.Refs) < 1 {
+		return o.info(0, "")
+	}
+
 	for i, refstr := range o.Refs {
-		ref, err := repo.ParseDatasetRef(refstr)
-		if err != nil {
+		if err := o.info(i, refstr); err != nil {
 			return err
 		}
+	}
 
-		if ref.IsPeerRef() {
-			printWarning(o.Out, "please specify a dataset for peer %s", ref.Peername)
+	return nil
+}
+
+// info pulls out the common info code so it can be re-run easily
```

Comment:
This comment should describe what the function does when executed, not what this pull request is doing to change the code, since the latter won't make sense when reading this in the future. Change it to something like "// info prints terse information about a single dataset"